backend/db.py

```diff
@@ -1,9 +1,9 @@
 from typing import Dict, List, Optional, cast
 from pymongo import MongoClient
-import os
 from datetime import datetime, timedelta
 import pytz # type: ignore
-import logging
+from config.env import ENVIRONMENT, MONGO_URI
+import custom_logger
```

Comment:
I think this should be aliased to just `logger`. `custom_logger` makes it seem like it's reserved for special use-cases. This goes for other occurrences as well.
```python
import custom_logger as logger
```